Documentation/k8s-pre-reqs.md

```diff
@@ -66,7 +66,34 @@ Environment="RKT_OPTS=--volume modprobe,kind=host,source=/usr/sbin/modprobe \
 ...
 ```
 
-Note that the kubelet also requires access to the userspace `rbd` tool that is included only in hyperkube images tagged `v1.3.6_coreos.0` or later.
+Note that the kubelet also requires access to the userspace `rbd` tool that is included only in hyperkube images tagged `v1.3.6_coreos.0` or later. See next section on how to deploy `rbd`.
+
+### Ceph and RBD utilities installed on the nodes
+
+The Kubernetes kubelet shells out to system utilities to mount Rook volumes. This means that every Kubernetes hosts must have these utilities installed. This requirement extends to the control plane, since there may be interactions between kube-controller-manager and the Ceph cluster.
+
+For Debian-based distros:
+
+```
+apt-get install ceph-fs-common ceph-common
+```
+
+For Redhat-based distros:
+
+```
+yum install ceph
+```
+
+For Other linux distro that dont have an explicit package manager, such as CoreOS, you can use a container with the ceph utilities deployed that you can leverage. To deploy the containers on your hosts, do the following:
+
+```
+$ cd /bin
```

Comment:
In a code block with multiple commands to run, I think it's better to not include the prompt sign (in this case $).  It makes copy/pasting much easier to not have to avoid the prompt sign each time.  In a multi command code block like this, I would remove the dollar sign.